kong/plugins/rate-limiting/schema.lua

```diff
@@ -69,6 +70,10 @@ return {
       if_field = "config.policy", if_match = { eq = "redis" },
       then_field = "config.redis_port", then_match = { required = true },
     } },
+    { conditional = {
```

Comment:
Let's add a test case covering this check (`spec/03-plugins/23-rate-limiting/01-schema_spec.lua`)

Reply:
I've wrote two tests, let me know if more tests is needed.